### DISCUSSION TITLE
Welcome to Nezuko Discussions!

### DISCUSSION THREAD

**Discussion by Ray-Senpai07** [EternityBots/Nezuko#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hikarirenthlei** [comment]
hii, I'm hikari
i have a question, how can i create a bot by eternity?
can u help me , do u have YouTube channel?
